gapic/templates/tests/unit/gapic/%name_%version/%sub/test_macros.j2

```diff
@@ -1855,12 +1817,12 @@ def test_{{ method_name }}_rest_no_http_options():
         {% else %}
         call.return_value = {{ method.output.ident }}()
         {% endif %}
-        client.{{ method_name }}(request={{ request }})
+        client.{{ method_name }}(request={{ request_dict }})
         {% endif %}{# is_async #}
 
         # Establish that the underlying gRPC stub method was called.
         call.assert_called()
-        _, args, _ = call.mock_calls[0]
+        _, args, kw = call.mock_calls[0]
```

Comment:
nit: the `kw` should be wrapped in `{% if routing_param %}` because otherwise it won't be used in the generated code. This could lead to linter warnings, right?

Reply:
Fixed in https://github.com/googleapis/gapic-generator-python/pull/2133/commits/5fa4a39059115ca0aaef17bbccd956045c938973